atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,47 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+    private final int maxTamanho = 10;
```

Comment:
Como é um valor constante poderia se reescrito assim:
```suggestion
    private final int MAX_TAMANHO = 10;
```